src/NativeShell/plugin.xml

```diff
@@ -19,32 +19,14 @@
   <platform name="android">
     <config-file target="res/xml/config.xml" parent="/*">
       <feature name="NativeShell">
-        <param name="android-package" value="org.jellyfin.mobile.NativeShell"/>
+        <param name="android-package" value="org.jellyfin.mobile.NativeShell" />
       </feature>
     </config-file>
     <config-file target="AndroidManifest.xml" parent="/manifest">
-      <uses-permission android:name="android.permission.WAKE_LOCK"/>
-      <uses-permission android:name="android.permission.BLUETOOTH"/>
-    </config-file>
-    <config-file target="res/values/strings.xml" parent="/*">
-      <string name="battery_optimizations_title">Disable Battery Optimizations</string>
-      <string name="battery_optimizations_message">Please disable battery optimizations for media playback while the screen is off.</string>
-      <string name="network_title">Allowed network types</string>
-      <string name="network_message">Do you want to allow the download to run over mobile data or roaming?\\nCharges may apply!</string>
-      <string name="wifi_only">WiFi only</string>
-      <string name="mobile_data">Mobile Data</string>
-      <string name="mobile_data_and_roaming">Mobile Data &amp; Roaming</string>
-      <string name="downloading">Downloading…</string>
-    </config-file>
-    <config-file target="res/values-de/strings.xml" parent="/*">
-      <string name="battery_optimizations_title">Batterieoptimierungen deaktivieren</string>
-      <string name="battery_optimizations_message">Bitte die Batterieoptimierungen deaktivieren um Medien mit ausgeschaltetem Bildschirm abgespielt werden können.</string>
-      <string name="network_title">Erlaubte Verbindungsmöglichkeiten</string>
-      <string name="network_message">Wollen Sie den Download über Mobile Daten oder Roaming elauben?\\nEs können Kosten anfallen!</string>
-      <string name="wifi_only">Nur WLAN</string>
-      <string name="mobile_data">Mobile Daten</string>
-      <string name="mobile_data_and_roaming">Mobile Daten &amp; Roaming</string>
-      <string name="downloading">Herunterladen…</string>
+      <uses-permission android:name="android.permission.WAKE_LOCK" />
+      <uses-permission android:name="android.permission.BLUETOOTH" />
+      <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+      <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
why do we need `read`?

Reply:
Permissions are a mess on Android and I don't trust vendor changes to operating systems for storage access. We will likely have to make several more changes to accomodate other versions and models as bugs roll in.